client/src/services/token.ts

```diff
@@ -1,27 +1,18 @@
-import { graphLinks } from '../graphHelper'
-
 //TODO: Make this context rather than class
 export class TokenService {
     private static token = localStorage.getItem('token') ?? ''
-    private static expirationDate = new Date(localStorage.getItem('expirationDate') as string) ?? new Date()
-    static timeToExpire: number = 60 * 60 * 1000 - 100 // 1 hour - 100ms buffer
```

Comment:
The logic of expire token need to be implemented somewhere, we either keep it here or  move it to BE now, as a cookie

Reply:
This is here:

https://github.com/adbahrani/MSGraphDashBoard/pull/25/files#diff-6aff8e6f59900a9cf0266ccb52fbbcb1c913f373577c21b23d9fcc4d59fa41e5R12

Reply:
we can add expiry time to cookie, this can be tackled as next task, did not think deeply about this.